CMakeLists.txt

```diff
@@ -237,23 +237,30 @@ find_package(Boost 1.74 REQUIRED COMPONENTS headers log)
 # gRPC version of 1.30.2, but don't `REQUIRE` since we want to have
 # another try via pkgconfig.
 
-find_package(gRPC 1.30.2 CONFIG QUIET)
+include(FindPkgConfig)
+
+find_package(protobuf 3.12.4 CONFIG)
```

Comment:
When doing the build against the hand-built version of gRPC in the focal Docker image, I had a little trouble where it could find the gRPC `.cmake` files but those in turn didn't correctly load the protobuf .cmake files. As a guess, I tried pre-finding `protobuf` with `find_package` and that fixed it. I actually think this is just better than it was before, because before we were keying off the presence or absence of `gRPC` via `find_package` and assuming the same would be true of `protobuf`, but that's not necessarily true.